installing/installing_nutanix/nutanix-failure-domains.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="nutanix-failure-domains"]
+= Fault tolerant deployments using multiple Prism Elements
+include::_attributes/common-attributes.adoc[]
+:context: nutanix-failure-domains
+
+toc::[]
+
+By default, the installation program installs control plane and compute machines into a single Nutanix Prism Element (cluster). To improve the fault tolerance of your {product-title} cluster, you can specify that these machines be distributed across multiple Nutanix clusters by configuring failure domains.
+
+A failure domain represents an additional Prism Element instance that is available to {product-title} machine pools during and after installation.
+
+include::modules/installation-nutanix-failure-domains-req.adoc[leveloffset=+1]
+
+== Installation method and failure domain configuration
+
+The {product-title} installation method determines how and when you configure failure domains:
+
+* If you deploy using installer-provisioned infrastructure, you can configure failure domains in the installation configuration file before deploying the cluster. For more information, see xref:../../installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc#installation-configuring-nutanix-failure-domains_installing-nutanix-installer-provisioned[Configuring failure domains].
++
+You can also configure failure domains after the cluster is deployed.
+* If you deploy using the {ai-full}, you configure failure domains after the cluster is deployed.
++
+For more information about configuring failure domains post-installation, see xref:../../post_installation_configuration/adding-nutanix-failure-domains.adoc#adding-failure-domains-to-an-existing-nutanix-cluster[Adding failure domains to an existing Nutanix cluster].
+
+* If you deploy using infrastructure that you manage (user-provisioned infrastructure) no additional configuration is required. You manually distribute control plane and compute machines across failure domains.
```

Comment:
Not sure on this one at the moment. The phrase _You manually distribute_ makes it sound like the UPI install does the distribution as part of the install. But, I'm not sure.  
```suggestion
* If you deploy using infrastructure that you manage (user-provisioned infrastructure) no additional configuration is required. You can manually distribute control plane and compute machines across failure domains.
```

Reply:
Interesting point. I clarified the statement to state, "After the cluster is deployed, you can manually distribute control plane and compute machines across failure domains."